### PR TITLE
templatized policies

### PR DESCRIPTION
**Pull request by ckandag**
Signed-off-by: ckandag <ckandaga@redhat.com>

### CODE REVIEW

**Comment by ycao56** [comment]
The PR looks to me assuming test cases will be added in a separate PR to cover various scenarios. Approving it and putting it on hold as I want @willkutler to take a second look before merging it.

**Review by willkutler — APPROVED** [review]

**Pull request merged**
